### PR TITLE
Fix the inconsistent response from API server (internal/public)

### PR DESCRIPTION
**Pull request by cdlliuy**
#155646346. See tracker issue description for the proposed changes

### CODE REVIEW

File: api/test/unit/routes/policies.test.js [hunk 1]
```diff
@@ -141,6 +145,7 @@ describe('Routing Policy Creation', function() {
           .set("Authorization", "fake-token")
           .send(fakePolicy)
           .end(function(error, result) {
+            console.log(result);
```
Comment (qibobo) [inline api/test/unit/routes/policies.test.js]: remove it?

File: api/app.js [hunk 2]
```diff
@@ -157,9 +155,7 @@ module.exports = function(settings, callback) {
     var errorResponse = {};
     if (err) {
       errorResponse = {
-        'success': false,
         'error': err,
-        'result': null
```
Comment (boyang9527) [inline api/app.js]: what is the reason why we have 'success' and 'result' defined in error message previously?

Comment (boyang9527) [reply api/app.js]: @qibobo  may be a question for you...

Comment (qibobo) [reply api/app.js]: @boyang9527 I did not change the result content of policy when I did the public api relevant PRs. It should be created in the origin commits of apiserver.

**Pull request merged**